crates/bevy_ui/src/layout/mod.rs

```diff
@@ -30,6 +30,14 @@ pub struct LayoutContext {
 }
 
 impl LayoutContext {
+    fn default() -> Self {
+        Self {
+            scale_factor: 1.0,
+            physical_size: Vec2::ZERO,
+            min_size: 0.0,
+            max_size: 0.0,
+        }
+    }
```

Comment:
This should be a `Default` trait impl.

Reply:
Yes, that make sense! Added those changes in the newest commit!

Reply:
No, as in `LayoutContext` should implement Default instead of a `default` method like this.

Reply:
```rust
impl Default for LayoutContext {
   fn default() -> Self {
        Self {
            scale_factor: 1.0,
            physical_size: Vec2::ZERO,
            min_size: 0.0,
            max_size: 0.0,
        }
   }
}
```

Reply:
Oh, I misunderstood your sugestion, thanks for clarifying! 
The last commit should already have those changes implemented.